scripts/rpm-upgrade.sh

```diff
@@ -179,8 +179,8 @@ fi
 
 check_mariadb_server_and_create_structures
 
-# Store information about server version and available plugins/engines before
-# upgrade
+# Store information about the server before upgrade
+collect_dependencies old
```

Comment:
`collect_dependencies_{old,new}` will run for both major / minor upgrades.
But in the case of a major upgrade, what we are doing with this information?

Seems that for now `reqs-* , ldd-*` files are only used in `check_upgraded_versions()` for `$test_type == "minor"`

Reply:
You are right, for now at least it doesn't make sense to collect it for major upgrades. Only instead of making it optional in 4 places, maybe it is better to make it optional within `collect_dependencies` itself.

Reply:
Yes, making it optional in `collect_dependencies `is better.

Reply:
done